pir/cpp/client.h

```diff
@@ -26,6 +26,8 @@ namespace pir {
 using ::private_join_and_compute::StatusOr;
 
 class PIRClient {
+  friend class PIRClientTest;
```

Comment:
Is this really necessary?

Reply:
Yes, otherwise the tests cannot access the decryptor. Without that we can only do end-to-end tests.